### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent.Incorrect - false positive with multiple arrays in argument list

### ISSUE THREAD

**Issue by VasekPurchart** [squizlabs/PHP_CodeSniffer#411]
I think this is a continuation of #337 and #358, I have found another failing situation (which was not reported in 1.5.x versions):

This code passes fine:
``` php
$this
	->method(array(
		'foo' => 'bar',
	), 'arg');

foo();
```
but when you add another array to the argument list, it gets confused:
``` php
$this
	->method(array(
		'foo' => 'bar',
	), 'arg', array(
		'foo' => 'bar',
	));

foo();
```
and reports error on the line with `foo();` (and in some cases on subsequent lines as well).

**Comment by gsherwood** [comment]
I've committed a fix for this case.
